src/DataBuilder.php

```diff
@@ -609,7 +611,12 @@ protected function getPerson()
     {
         $personData = $this->person;
         if (!isset($personData) && is_callable($this->personFunc)) {
-            $personData = call_user_func($this->personFunc);
+            try {
+                $personData = call_user_func($this->personFunc);
+            } catch (PersonFuncException $exception) {
```

Comment:
Doesn't this only handle the specific case of someone throwing this particular type of exception? That doesn't seem to handle the general case of errors happening in user code. It seems like we should swallow all exceptions regardless of type coming from user code.